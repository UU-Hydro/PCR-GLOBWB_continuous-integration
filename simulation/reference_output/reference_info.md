# Reference models

## release_v2.4.0_alpha
https://github.com/UU-Hydro/PCR-GLOBWB_model/releases/tag/v2.4.0_alpha
https://github.com/UU-Hydro/PCR-GLOBWB_model/commit/0511485ad3ac0a1367d9d4918d2f61ae0fa0e900
