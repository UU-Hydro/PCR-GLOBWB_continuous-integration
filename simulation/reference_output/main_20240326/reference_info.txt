https://github.com/UU-Hydro/PCR-GLOBWB_model/commit/ccf2bf9e83ffd50448a530cb34864546d62d9ac8